nuxeo-retention/src/main/resources/OSGI-INF/retention-vocabularies.xml

```diff
@@ -2,16 +2,16 @@
 <component name="org.nuxeo.retention.vocabularies" version="1.0">
   <extension target="org.nuxeo.ecm.directory.GenericDirectory" point="directories">
 
-    <directory name="template-l10nvocabulary" template="true" extends="template-directory">
+    <directory name="template-vocabulary" template="true" extends="template-directory">
```

Comment:
his whole `<directory ...` statement should not be needed anymore, it is already defined as is in https://github.com/nuxeo/nuxeo/blob/master/modules/platform/nuxeo-platform-default-config/src/main/resources/OSGI-INF/nxdirectories-contrib.xml#L8